doc/man5/flux-config-job-manager.rst

```diff
@@ -83,6 +87,84 @@ release-after
   have completed housekeeping when the timer fires are released. Following
   that, resources are released as each execution target completes.
 
+.. _plugin_specific_keys:
+
+PLUGIN SPECIFIC KEYS
+====================
+
+The following configuration keys are supported by included jobtap plugins.
+The documented config will have no effect if the associated plugin is not
+loaded into the job-manager.
+
+perilog.so
+----------
+
+The ``perilog.so`` plugin supports configuration and execution of job-manager
+prolog and/or epilog. The following keys are supported when the ``perilog.so``
+plugin is loaded:
+
+prolog
+   (optional) Table of configuration for a job-manager prolog. If enabled,
+   the prolog is initiated when the job enters the RUN state before any
+   job shells (i.e. user processes) are started. The following keys are
+   supported for the ``[job-manager.prolog]`` table:
+
+   command
+      (optional) An array of strings specifying the command to run. If
+      ``exec.imp`` is set, the the default command is ``["flux-imp",
+      "run", "prolog"]``, otherwise it is an error if command is not set.
+   per-rank
+      (optional) By default the job-manager prolog only runs ``command``
+      on rank 0. With ``per-rank=true``, the command will be run on each
+      rank assigned to the job.
+   timeout
+      (optional) A string value in Flux Standard Duration specifying a
+      timeout for the prolog, after which it is terminated (and a job
+      exception raised). The default prolog timeout is 30m. To disable
+      the timeout use ``0`` or ``infinity``.
+   kill-timeout
+      (optional) Floating-point number of seconds to wait after sending
+      SIGTERM to send SIGKILL to the prolog when the prolog has been
+      canceled due to timeout or exception.
```

Comment:
Should `kill-timeout` be an FSD to match `timeout`?

Reply:
It could, but is mainly there for testing and it seemed like a bit of wasted effort because then it would require extra tests, etc. However, it is pretty simple if you feel strongly about it. Perhaps more important would be a note saying this parameter is mainly meant for testing?